uwg/RSMDef.py

```diff
@@ -74,314 +105,323 @@ def __init__(self,lat,lon,GMT,height,T_init,P_init,parameter,z_meso_path):
         # Define self.nz0, self.nzref, self.nzfor, self.nz10, self.nzi
         for iz in range(len(self.z_meso)-1):
             # self.nz0: self.z index >= reference height for weather station
-            eq_th = self.is_near_zero(self.z[iz] - parameter.tempHeight)
-            if (eq_th == True or self.z[iz] > parameter.tempHeight) and ll==True:
-                self.nz0 = iz+1   # layer number at zmt (m)
+            eq_th = is_near_zero(self.z[iz] - parameter.tempHeight)
+            if (eq_th is True or self.z[iz] > parameter.tempHeight) and ll is True:
```

Comment:
Yeah I know... I just never really liked this PEP8 recommendation because I think it causes confusion when you combine it with Python's dynamic typing.  

For example if greeting wasn't a boolean. 
```
greeting = 1
if greeting: print('x')
>> x
```
And there's equivalent versions for lists, None and other types. So my instinct is to explicitly define what's being tested in an if/else condition so I don't have to hunt around looking for what the variable type is when rereading/editing code. 

BUT, it's not a big deal, so I'll change it to the implicit version.

Reply:
Maybe it also helps to know that PEP8 only recommends using `if x` and `if not x` in the case that `x` is a boolean.  You are not supposed to use it in the case that it's a None (for that you should be using `if x is None`).  And you also shouldn't use it for the case that `x` is a number.  So you always know the type is a boolean if you're following PEP8.

Reply:
@chriswmackey ah! Finally that makes sense to me.